### PR TITLE
Release @adeira/bc-checker 0.1.0

### PR DESCRIPTION
**Pull request by tbergq**

### CODE REVIEW

**Comment by mrtnzlml** [comment]
@tbergq I tried to resolve the conflicts via GH UI but it's still not happy. Could you please check it locally?

**Comment by tbergq** [comment]
@mrtnzlml 🚀

File: scripts/publishNPMPackages.js [hunk 1]
```diff
@@ -32,6 +32,7 @@ import { invariant } from '@adeira/js';
       '@adeira/flow-bin',
       '@adeira/eslint-config',
       '@adeira/global-id',
+      '@adeira/bc-checker',
```
Comment (mrtnzlml) [inline scripts/publishNPMPackages.js]: Alphabetical order would be perhaps nicer... :)

Comment (tbergq) [reply scripts/publishNPMPackages.js]: 🙈

**Pull request merged**